oger/doc/txt.py

```diff
@@ -141,3 +142,17 @@ def iter_documents(self, source):
                     continue  # member is not a regular file
                 id_ = basename(member.name)
                 yield self._document(stream, id_)
+
+
+class TXTTSVLoader(DocIterator, _TXTLoaderMixin):
+    '''
+    Loader for single TSV document consisting of 
+    ColumnA -> [id]
+    ColumnB -> [text] format
+    '''
+    def iter_documents(self, source):
+        with open(source, 'r') as f:
+            data = csv.reader(f, dialect='excel', delimiter='\t')
+            next(data) # skips header row
+            for idx, row in data:
+                yield self._document(row, idx)
```

Comment:
```suggestion
            for id_, text in data:
                yield self._document([text], id_)
```
`self._document` expects an iterable of lines. It might work with a single string because it splits and rejoins the characters, but it's a bit brittle.
`idx` sounds like `index`, but this value is realy the document ID.